ServiceStack/src/ServiceStack/Validation/ValidationFeature.cs

```diff
@@ -251,9 +256,36 @@ public virtual async Task ValidateRequestAsync(object requestDto, IRequest req,
             ? await validator.ValidateAsync(validationContext, token)
             // ReSharper disable once MethodHasAsyncOverloadWithCancellation
             : validator.Validate(validationContext);
-            
-        if (!result.IsValid)
+        
+        if (TreatInfoAndWarningsAsErrors && result.IsValid)
+        {
+            return;
+        }
+        
+        if (!result.IsValid && result.Errors.Any(x => x.Severity == Severity.Error))
+        {
             throw result.ToWebServiceException(requestDto, this);
+        }
+            
+        var ver = new ValidationErrorResult();
```

Comment:
This was the only way I could get the Meta property populated. Perhaps there is a better option?